app/scripts/directives/mobileServiceInstanceRow.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('mobileServiceInstanceRow', {
+    controller: [
+      '$filter',
+      '$routeParams',
+      '$scope',
+      'APIService',
+      'AuthorizationService',
+      'BindingService',
+      'Constants',
+      'DataService',
+      'ListRowUtils',
+      'ServiceInstancesService',
+      MobileServiceInstanceRow
+    ],
+    controllerAs: 'row',
+    bindings: {
+        apiObject: '<',
+        serviceClass: '<',
+        binding: '<',
+        state: '<'
+    },
+    templateUrl: 'views/_mobile-service-instance-row.html'
+  });
+
+  function MobileServiceInstanceRow($filter,
+                              $routeParams,
+                              $scope,
+                              APIService,
+                              AuthorizationService,
+                              BindingService,
+                              Constants,
+                              DataService,
+                              ListRowUtils,
+                              ServiceInstancesService) {
+    var row = this;
+    var serviceInstanceDisplayName = $filter('serviceInstanceDisplayName');
+    row.serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
+    row.state = row.state || {};
+    var annotationSpace = "org.aerogear.binding." + this.apiObject.metadata.name;
+    var watches = [];
+    _.extend(row, ListRowUtils.ui);
+
+    $scope.$on('$destroy', function(){
+        DataService.unwatchAll(watches);
+    });
+
+    watches.push(DataService.watchObject(Constants.MOBILE_CLIENT_VERSION, $routeParams.mobileclient, {namespace: $routeParams.project}, function(mobileClient, action) {
+        row.mobileClient = mobileClient;
+        row.annotations = [];
+        row.parameterData = {CLIENT_ID: row.mobileClient.metadata.name};
+        _.each(row.mobileClient.metadata.annotations, function(annotation, name){
+            if(name.split("/")[0] === annotationSpace){
+                row.annotations.push(JSON.parse(annotation));
+            }
+        });
+    }));
+
+    var getServicePlan = function() {
+      var servicePlanName = ServiceInstancesService.getServicePlanNameForInstance(row.apiObject);
+      return _.get(row, ['state', 'servicePlans', servicePlanName]);
+    };
+
+    row.$doCheck = function() {
+      row.servicePlan = getServicePlan();
+      row.displayName = serviceInstanceDisplayName(row.apiObject, row.serviceClass);
+      row.isBindable = BindingService.isServiceBindable(row.apiObject, row.serviceClass, row.servicePlan);
```

Comment:
A mobile service is also identified as bindable to a client if the service class has the tag `mobile-client-enabled`. It might be an easier check.